Source/CFString.c

```diff
@@ -1641,6 +1646,25 @@ CFStringFold (CFMutableStringRef str, CFOptionFlags flags, CFLocaleRef locale)
     CFStringCaseMap (str, locale, flags, _kCFStringFold);
 }
 
+#ifdef __UNORM2_H__
```

Comment:
Please remove support for unorm altogether. This should simplify the code significantly.